services/influxdb/values-squash-sandbox.yaml

```diff
@@ -0,0 +1,257 @@
+influxdb:
+  ## influxdb image version
+  ## ref: https://hub.docker.com/r/library/influxdb/tags/
+  image:
+    repository: "influxdb"
+    tag: "1.8.2-alpine"
+    pullPolicy: IfNotPresent
```

Comment:
Same question here about whether the upstream Helm chart will handle this for us.

Reply:
See above.